### ISSUE TITLE
Layout/FirstArgumentIndentation is off by one when the method is preceded by a `*`

### ISSUE THREAD

**Issue by pdobb** [rubocop-hq/rubocop#9115]
I am experiencing an off-by-1 disagreement with the Layout/FirstArgumentIndentation cop (configured with `EnforcedStyle: consistent_relative_to_receiver`) when a leading `*` is involved. But preceding dots and `&.` behave fine.

```ruby
# Valid
[
  value1,
  my_object
    .my_method(
      arg1)  # 2 spaces from start of `.my_method`
]

# Valid
[
  value1,
  my_object
    &.my_method(
      arg1)  # 2 spaces from start of `&.my_method`
]

# Invalid :(
[
  value1,
  *my_method(
    arg1)  # 2 spaces from start of `*my_method`
]
```

--------

## Expected behavior

Layout/FirstArgumentIndentation shouldn't auto-correct to and shouldn't require an extra space when an `*` precedes the method call.

## Actual behavior

```ruby
# This is considered invalid, when it shouldn't be
[
  value1,
  *my_method(
    arg1,  # 2 spaces from start of `*my_method`
    arg2)
]

temp.rb:23:5: C: Layout/FirstArgumentIndentation: Indent the first argument one step more than my_method(.
    arg1,
    ^^^^

# And will autocorrect to this, when it shouldn't (Note the extra / 3rd space before `arg1`
[
  1,
  *my_method(
     arg1,  # 3 spaces from start of `*my_method`
    arg2)
]
```

## Steps to reproduce the problem

Just add this to a ruby file and run the Layout/FirstArgumentIndentation cop

```ruby
[
  value1,
  *my_method(
    arg1)
]
```

Note, that my configuration is:

```YAML
Layout/FirstArgumentIndentation:
  EnforcedStyle: consistent_relative_to_receiver
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ rubocop -V
1.4.2 (using Parser 2.7.2.0, rubocop-ast 1.2.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-performance 1.9.0
  - rubocop-rails 2.8.1
  - rubocop-rake 0.5.1
```
